dev/com.ibm.ws.microprofile.metrics.private/resources/WEB-INF/web.xml

```diff
@@ -19,17 +19,17 @@
     <servlet-class>com.ibm.ws.microprofile.metrics.privateapi.PrivateMetricsRESTProxyServlet</servlet-class>
     <load-on-startup>1</load-on-startup>
   </servlet>
-  
+
```

Comment:
I know for 18.0.0.3 mp metrics added a config flag (<mpMetrics authentication="false|true" />) that allows a user to turn off authentication.  Not familiar with the implementation of that, but it would be good to make sure that the flag works correctly with the viewer role as well

Reply:
I believe the test flow in MetricsAuthenticationTest covers this. Specifically, it tests accessing the metrics with Administrator, Viewer and then sets the config flag to false and accesses as unauthenticated.